src/klein/_dihttp.py

```diff
@@ -92,7 +92,7 @@ def registerInjector(
 
     def injectValue(self, instance, request, routeParams):
         # type: (Any, IRequest, Dict[str, Any]) -> DecodedURL
-        return request.getComponent(self.interface)
+        return cast(DecodedURL, request.getComponent(self.interface))
```

Comment:
Fixes Mypy error:
```console
src/klein/_dihttp.py:95:9: error: Returning Any from function declared to return "DecodedURL"  [no-any-return]
```

Reply:
Note Hyperlink 20 adds type hints, so it's likely related to that, though I'm not sure why Mypy didn't complain before; seems like it should have.